engine/engine.py

```diff
@@ -93,12 +93,14 @@ def fit(self, epoch, is_training=True):
                 #forward call
                 #output is a namespace with members as added in the forward call
                 #and subsequently used in loss()
+                fwd_output=self._model(input_data)
+                """
                 try:
-                    fwd_output=self._model(input_data)
```

Comment:
Hey @abhishekabhishek, this is put in so one of the models stays compatible. Did you remove this for a particular reason?

Reply:
Yes. It suppresses all errors from forward even those relevant to other implementations. This was a cheap workaround but should modify this to catch error specific to conditional-VAE

Reply:
It was a cheap fix in the first place. Will modify the conditional VAE accordingly. 👍